backend/src/test/kotlin/net/matsudamper/mastodon/rss/FakeRepositories.kt

```
@@ -171,3 +183,78 @@ class FakeNoteRepository : NoteRepository {
+        val index = stored.indexOfFirst { it.id == id }
+        if (index == -1) return
+        val current = stored[index]
+        stored[index] = current.copy(title = title, siteUrl = siteUrl, format = format)
+    }
+
+    override fun recordFetchSuccess(
+        id: FeedId,
+        fetchedAt: Instant,
+        validators: FeedFetchValidators,
+    ) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)